src/utils/error/handler.js

```diff
@@ -2,17 +2,24 @@ const errors = require("./errors");
 const httpStatus = require("../httpStatus");
 
 module.exports = (err, req, res, next) => {
-  
-  const { status, message } = errors.getErrorInfo(
-    err.name || err.message
-  );
+  let statusMessage;
+  let status;
+  let message;
+  let retval = { status, message };
 
-
-  const statusMessage = httpStatus.getStatusText(status);
-
-  res.status(status).send({
-    error: message,
+  if (err.name === "MongoError") {
+    retval.status = httpStatus.BAD_REQUEST;
```

Comment:
Not all MongoError should be bad request. Ths error is a general mongo error. You will need to check the Error Code field for this case

Reply:
There are validation errors from mongoose that has error name validation error, so I need a whole new file for all of those repo errors essentially

Reply:
Will probably withdraw pr until I get that file finished